encoding/av1-break-even-calculator/index.html

```diff
@@ -124,31 +116,40 @@ <h3 class="my-3">Calculation multipliers</h3>
 <div class="container">
   <div class="row">
     <table class="table">
-      <thead>
+      <!-- <thead>
```

Comment:
Do we think we will need this again in the near future or can we just remove the code completely ?